src/otlp.rs

```diff
@@ -36,6 +38,10 @@ where
     let exporter: SpanExporterBuilder = match protocol.as_str() {
         "http/protobuf" => opentelemetry_otlp::new_exporter()
             .http()
+            .with_http_client(HyperClient::new_with_timeout(
+                hyper::Client::new(),
+                Duration::from_millis(1500), // TODO: make configurable
```

Comment:
@spurge any idea what value should be used here?

Reply:
Good question! I don't know. As it's a Lambda, it shouldn't matter, as it will export everything before shutdown. [Three seconds is default in @viaplay/telemetry-node](https://github.com/nentgroup/telemetry-node/blob/b83a8ea695e709d6af512173a066d7c745f58957/src/init.ts#L85). [In telemetry-go it's a minute](https://github.com/nentgroup/telemetry-go/blob/b92bc6109394bac40ea9371b8eccbe18b1b8c946/telemetry/config.go#L157) :)

Reply:
This timeout is between our rust service and the local extension or local collector container

Reply:
Exporter timeout follows this spec:

<img width="675" alt="Screenshot 2024-02-01 at 15 00 53" src="https://github.com/nentgroup/telemetry-rust/assets/1128370/039aba1e-326f-4c0e-b527-8ed5fbea41e4">

Reply:
I don't understand why http client needs it's own timeout, perhaps I should just reuse exporter timeout value here

Reply:
I re-used exporter timeout here